### ISSUE TITLE
[BUG] Microphone icon gone/missing in v40 of extension

### ISSUE THREAD

**Issue by Scotty-Trees** [kgshank/gse-sound-output-device-chooser#197]
Greetings,

Love your extension very much btw been using it for a long time!

**Describe the bug**
So upon upgrading the extension to v40 just very recently, the icon for the microphone no longer shows in the top panel. I'm unsure if this is by design or a bug, but I just wanted to bring it to your attention. In previous versions, the microphone icon would be in the top panel next to the sound volume icon. I found it rather handy since I could take one quick glance to see if my microphone was on/off without even having to click anything, so for me that was super simple, but also super useful. 

**Extension Information**
* gnome-shell v 41.1
* Extension v40
* Installed via Gnome Extension website
* EndeavourOS running Gnome 41.1

Here's how the top panel looked before, showing the microphone icon. It's greyed out because I use the papirus dark icon theme, and means my microphone is at 0% volume, if I moved the mic volume up to a higher level, the icon would turn white like the others, fyi.
![menu-sound-mic-missing](https://user-images.githubusercontent.com/45135187/141821047-97cc2c93-4889-4d7f-a84c-ce8a2982a5f9.png)

Here's how the top panel looks now on v40, with the mic icon missing/gone.
![menu-sound-mic-NOT-missing](https://user-images.githubusercontent.com/45135187/141821068-401abae5-7291-44fb-9c7f-fb697d321ad3.png)

Like I mentioned I don't know if this is intentional or a bug, so feel free to let me know or correct me, no worries. Also, I'll leave my sound settings configuration here if that is helpful below. Let me know if there's any other info/logs you may need from me, I'm very happy to help out anyway I can. Thank you for your time!

![Screenshot from 2021-11-15 11-50-43](https://user-images.githubusercontent.com/45135187/141821925-1555202d-9ba3-4060-9417-b38dc3eca00e.png)

**Comment by thesmallcreeper** [comment]
#174 sus... :eyes: 
Well yeah I did that change (sorry :heart:) in order for mic indicator being shown only when mic is being used by a application (other than gnome-settings).
If you want you can request the all the way around as a feature :)

**Comment by Scotty-Trees** [comment]
@thesmallcreeper Yeah no worries no need to be sorry! I know the mic icon used to be hidden in previous versions of Gnome and I did used to like it that way at the time. But in newer Gnome versions they added the mic icon in the top panel, and honestly I got used to always seeing it and actually preferred it that way oddly enough. Like I mentioned I find it valuable and helpful to be able to see if my mic is on/off at just a glance in the top panel. Sometimes I'll have my mic turned on and other times I'll have it off OR I forget to turn it off and having a quick glance to see real quick is helpful to my workflow. I think I shall make a request to add it, but you don't have to consider this comment a formal request, I can post a new topic for that! Thanks again for the quick reply and for making what I'd argue is a necessary extension in Gnome ;)
